Anime-App/Feature/Anime/View/ViewControllers/AnimeViewController.swift

```diff
@@ -102,12 +114,35 @@ extension AnimeViewController: UISearchBarDelegate {
 
 extension AnimeViewController {
     
+    func addHapticFeedBack() {
+        let hapticFeedbackIsOn = UserDefaults.standard.bool(forKey: "UseHaptic")
+        if hapticFeedbackIsOn {
+            print("haptic feedback is on")
+            let generator = UIImpactFeedbackGenerator(style: .heavy)
+            generator.impactOccurred()
+        }
+    }
+    
+    @objc func addSound() {
```

Comment:
A good habit to do is before you play a sound is to stop any sounds that are playing, you don't want to do this in an if statement it should be outside and just the first thing you do. This is to stop sounds from overlapping and playing over one another